linera-views/src/log_view.rs

```diff
@@ -65,13 +65,13 @@ where
     }
 
     fn rollback(&mut self) {
-        self.was_cleared = false;
+        self.delete_storage_first = false;
         self.new_values.clear();
         *self.hash.get_mut() = self.stored_hash;
     }
 
     fn flush(&mut self, batch: &mut Batch) -> Result<(), ViewError> {
-        if self.was_cleared && self.stored_count > 0 {
+        if self.delete_storage_first && self.stored_count > 0 {
             batch.delete_key_prefix(self.context.base_key());
             self.stored_count = 0;
         }
```

Comment:
I suspect the same trick `self.stored_hash = None` works here then we can simplify below and remove the comment.

Reply:
Ok, done.